integration/integration_test.go

```diff
@@ -5225,21 +5225,22 @@ func testBPFSessionDifferentiation(t *testing.T, suite *integrationTestSuite) {
 		// Signal that the client has finished the interactive session.
 		doneCh <- true
 	}
-	writeTerm(termA)
-	writeTerm(termB)
+	go writeTerm(termA)
```

Comment:
If we `go` here then we can't call require.NoErrors (or any require functions) inside writeTerm, as it's an error to call t.Fatal outside the test goroutine. Please change those to assert or t.Errorf.